analyzer/src/main/kotlin/managers/GoMod.kt

```diff
@@ -191,6 +194,22 @@ class GoMod(
         return graph
     }
 
+    /**
+     * Return the list of all modules contained in the dependency tree with resolved versions.
+     */
+    private fun getAllModules(projectDir: File): Set<Identifier> {
+        val result = mutableSetOf<Identifier>()
+
+        for (line in run("list", "-m", "all", workingDir = projectDir).requireSuccess().stdout.lines()) {
```

Comment:
`run` already calls `requireSuccess` so you don't need to do that explicitly.

Style-wise, I'd prefer to introduce a variable and use `forEach`:

```
val list = run("list", "-m", "all", workingDir = projectDir)
list.stdout.lines().forEach { line ->
  // ...
}
```